### ISSUE TITLE
bConnect does not work in PS 6.x and higher

### ISSUE THREAD

**Issue by domushnik** [baramundisoftware/PS-bConnect#53]
**Prior to placing the issue, please check following:** *(fill out each checkbox with an `X` once done)*
- [x] I have understood that answers are voluntary and community-driven, and not commercial support.
- [x] I have verified that my issue has not been already answered in the past. I also checked previous [issues](https://github.com/baramundisoftware/PS-bConnect/issues).

**Describe the bug**
When I import the bConnect module in PS 7.x I get the following error:
```
(4,40): error CS0246: The type or namespace name 'ICertificatePolicy' could not be found (are you missing a using directive or an assembly reference?)
public class ignoreCertificatePolicy : ICertificatePolicy {
```
According to the MS article [ServicePointManager.CertificatePolicy Property](https://docs.microsoft.com/en-us/dotnet/api/system.net.servicepointmanager.certificatepolicy?view=netframework-4.8), the property CertificatePolicy in the class ServicePointManager is obsoleted. 

**To Reproduce**
1. Run Import-Module bConnect in PS 6.x or higer

**Expected behavior**
The module should not contain any obsoleted properties.

**System information**
| Question | Answer |
| --- | --- |
| My operating system | Win10 |
| baramundi Server Version | 2020R1 |
| PS-bConnect Version | 20.1.0.5 |

**Comment by T0biii** [comment]
in the .psd file PowerShell Version 3.0 is specified 
https://github.com/baramundisoftware/PS-bConnect/blob/9730f097bd55cdedf3f0160bbdd3226c3111c29f/bConnect/bConnect.psd1#L9
maybe the obsoleted properties are not the only blocker für PowerShell 6.x?

**Comment by T0biii** [comment]
@domushnik I tested the Get-bConnectInfo command:
![image](https://user-images.githubusercontent.com/5702338/149771492-41e48b8f-a691-42fe-bbe6-7b2de1b9beb8.png)
but i got an the same error at the import

**Comment by domushnik** [comment]
@T0biii , yes. Also, it makes impossible to ignore the server certificate.

**Comment by T0biii** [comment]
> @T0biii , yes. Also, it makes impossible to ignore the server certificate.

if you have any idea how to fix it for PS 6.x and still got support for PowerShell 3.x?
PRs are welcome

**Comment by DanielK37** [comment]
Although this thread is somewhat old, I would add my experience from just a few minutes ago: I run PS version 7.4.2 on Windows 10 and at first when trying to import this module I also received this message (that is actually why I landed on this page):

`(4,40): error CS0246: The type or namespace name 'ICertificatePolicy' could not be found (are you missing a using directive or an assembly reference?)
public class ignoreCertificatePolicy : ICertificatePolicy {`

But when I did run the import a second time several minutes later it ran successful and I was able to initialize a connection to our baramundi server.

**Comment by NickETH** [comment]
This problem can be circumented by using the #Requires statement at very first line in the script. See:
https://learn.microsoft.com/th-th/powershell/module/microsoft.powershell.core/about/about_requires?view=powershell-5.1

Example:
#Requires -Version 5.1
